Apps/Admin/Tests/Functional/cypress/integration/e2e/agentaccess.cy.js

```diff
@@ -50,62 +46,51 @@ describe("Provision", () => {
+        cy.get("[data-testid=create-btn]").click();
         cy.get("[data-testid=provision-dialog-modal-text]").should(
             "be.visible"
         );
-        cy.get("[data-testid=username-input]").clear().type("FncTstUser1");
-        cy.get("[data-testid=identity-provider-select]").click({
-            force: true,
-        });
-        cy.get("[data-testid=identity-provider]")
-            .contains("IDIR")
-            .click({ force: true });
-        cy.get("[data-testid=roles-select]").click({
-            force: true,
-        });
-        cy.get("[data-testid=role]")
-            .contains("AdminUser")
-            .click({ force: true });
-        cy.get("[data-testid=save-btn]").click({ force: true });
+        cy.get("[data-testid=username-input]").clear().type(user);
+        cy.get("[data-testid=identity-provider-select]").click({ force: true });
+        cy.get("[data-testid=identity-provider]").contains("IDIR").click();
+        cy.get("[data-testid=roles-select]").click();
+        cy.get("[data-testid=role]").contains("AdminUser").click();
+        cy.get("[data-testid=save-btn]").parent().parent().click(0, 0);
+        cy.get("[data-testid=save-btn]").click();
 
         cy.log("Validate duplicate user error.");
-
         cy.get("[data-testid=add-error-alert]").should("exist");
-        cy.get("[data-testid=cancel-btn]").click({ force: true });
+        cy.get("[data-testid=cancel-btn]").click();
         cy.get("[data-testid=provision-dialog-modal-text]").should("not.exist");
+    });
 
-        cy.log("Edit user.");
-
-        cy.get("[data-testid=query-input]").clear().type("fnct");
-        cy.get("[data-testid=search-btn]").click({ force: true });
+    it("Edit User", () => {
+        cy.get("[data-testid=query-input]").clear().type(user);
+        cy.get("[data-testid=search-btn]").click();
         cy.get("[data-testid^=agent-table-username-]")
-            .contains("fnctstuser1")
+            .contains(user.toLowerCase())
             .parents(".mud-table-row")
             .get("[data-testid^=agent-table-edit-btn]")
-            .click({ force: true });
+            .click();
         cy.get("[data-testid=provision-dialog-modal-text]").should(
             "be.visible"
         );
-        cy.get("[data-testid=roles-select]").click({
-            force: true,
-        });
-        cy.get("[data-testid=role]")
-            .contains("AdminAnalyst")
-            .click({ force: true });
-        cy.get("[data-testid=save-btn]").click({ force: true });
+        cy.get("[data-testid=roles-select]").click();
+        cy.get("[data-testid=role]").contains("AdminAnalyst").click();
+        cy.get("[data-testid=save-btn]").parent().parent().click(0, 0);
+        cy.get("[data-testid=save-btn]").click();
         cy.get("[data-testid=provision-dialog-modal-text]").should("not.exist");
 
         cy.log("Validate user edit.");
-
+        const rowSelector = "[data-testid=agent-table] tbody tr.mud-table-row";
```

Comment:
NIT: Considering the variable was once used to reduce the duplication of the selector, this seems to go back on that decision making the variable unnecessary, shouldn't this be defined in a higher scope for all the tests to use, or just define the selector directly in the get method:

```TypeScript 
cy.get("[data-testid=agent-table] tbody tr.mud-table-row")
````

Reply:
As discussed, I'll leave this as is as the usage is low.

Reply:
Agreed - this should be moved outside the test definitions.